src/color-slider/color-slider.js

```diff
@@ -60,13 +60,14 @@ const Self = class ColorSlider extends NudeElement {
 	}
 
 	propChangedCallback ({name, prop, detail: change}) {
+		// Spinner values when tooltip is "progress"
+		let values = { min: 1, max: 100, step: 1, value: +(this.progress * 100).toPrecision(4) };
```

Comment:
We should not calculate this no matter what, but inside relevant conditionals.

Reply:
Agreed. My motivation was to make the code a bit DRYer. Fixed.